cmd/monaco/dependencygraph/dependencygraph.go

```diff
@@ -74,6 +76,18 @@ func writeGraphFiles(fs afero.Fs, manifestPath string, environmentNames []string
 			return fmt.Errorf("failed to encode dependency graph to DOT for environment %q: %w", e, err)
 		}
 		file := filepath.Join(folderPath, fmt.Sprintf("dependency_graph_%s.dot", e))
+
+		exists, err := afero.Exists(fs, file)
+		if err != nil {
+			return fmt.Errorf("failed to validate if output file %q already exists: %w", file, err)
+		}
+		if exists {
```

Comment:
Why did you decide to only append the date if the file already exists? Why not always append it?

Reply:
We do append the timestamp always on the created download folder (except if we pass an argument setting the folder).
Since this command will probably be used only by a few customers, I think we can let it be as is.

Reply:
On the folder yes, on the containing files with fixed names only if they exist.
Unlike the download folder, where the timestamp tells me "this is what I downloaded on {date}", there's no inherent value of having the timestamp in these files or the download's manifest.yaml. 

Thus I really think appending it only if needed makes more sense in this case, even if it means an extra if in the code